### ISSUE TITLE
Reward account refreshing has a lag

### ISSUE THREAD

**Issue by Fell-x27** [cardano-foundation/cardano-wallet#2063]
# Context
 
When you send some funds+rewards your UTxO and reward balances don't get updated at the same time.

| Information  | -                                                     |
| ---          | ---                                                   |
| Version      | 2020.8.3 (git revision: b5b292b5a9f0ddccc0fc5d9e98bf3bf80f1a9d13) |
| Platform     | Linux          |
| Installation | From Hydra           |

# Steps to Reproduce

1. Do some transaction with withdrawal='self'
2. Check balance immediately

## Expected behavior

UTxO and reward balances have new values.

## Actual behavior
 
UTxO balance has a new value since transaction has been formed, but reward one gets changed only after confirmation.

---

**Comment by KtorZ** [comment]
Thanks for reporting @Fell-x27 :) 
This one is known and a duplicate of #1955 (see last-ish comment).
